### ISSUE TITLE
to daelus

### ISSUE THREAD

**Issue by dreammed** [Nulled-Daelus/SevenKnightsAI#18]
should we  add version number to interface?

**Comment by Nulled-Daelus** [comment]
Done.
I've added it to the log and to the status bar.
Version number is automatically pulled from the "Assembly" version number if you go to properties and look under application then Assembly Information. And will append the date of the build If we need more accurate then that then... We can go further I just cut it at that for now.

Not sure why we changed the logo though lol. Personally I always find it annoying when the icon changes.

**Comment by dreammed** [comment]
i just want up date to tend 
i try find somewhere to detect star of hero for add new function 
like sell hero ,sell item and replace hero with priority with star
any advice? and i think loot adventure scene hard to detect 
it a lot of condition eg. got 1star Hero ,2star Hero ,def item,matk item ,gem,accessory
i think i will don't check for this scene  and mix it all for one call adventure_loot
what do u think?

**Comment by Nulled-Daelus** [comment]
Yeah not entirely sure there. There's a lot of information we could gather but is it worth it? 

I have not been concerned with that window for collecting data as I'm not really sure there's much point other then for summary reasons...

**Comment by dreammed** [comment]
yeah i think so 
i try to update hero count when found loot hero scene 
but loop always find the scene i don't want ai always find scene 
i plan enchant main loop more accurate scene search

**Comment by Ryouichi** [comment]
I just read at Heartcold facebook post

"HEARTCOLD WAS HONORED TO ATTEND A PRIVATE Q&A SESSION WITH THE CEO OF NETMARBLE."

Hello. When Saturday, July 23, 2559 that I, as a representative of heartcold. Attending a meeting of the game seven, for arranging a great hotel with squat Bangkok by after the job done. There's a admin group things including I set up a small group with Executive Netmarble 4 to ask question about

I'll come by, stay tuned to read about what to do with us. Everyone. I have a chance to ask privately, with the highest executives of netmarble about this.

✓ Using the seven, ai by heartcold is fine. He used the word " it's " OK " and explain the hook if it don't destroy yourself any game is available.

✓ But even so, he said, "witty but the game already have an ai" something like that will have to go. Since in the game.

✓ Executive Max still explain that one game that has a flat, the player to use a different kind of project to keep not seven, ai by heartcold (prefer not to mention the name) as well as pro cheat numbers and different values.

Which, by the history and heartcold has never been a user banned. So assume that if you want to use the program to help play one of those seven, ai by heartcold is safe to use the best

Join the conversation in this time, we intend to look for an update from executive, ALL 4 of you to prepare for us, too. and it is an honor to ask answer me like a blood type, and the chance to ask about something privately. So what So, here it is.. More details will not be able to hear. :D

I still have the whole picture. PHOTO WITH RUN SIGNED ALL 4 of you... As evidence. Don't give are exposed. As well as other details as long as it's the duty of admin group seven, etc.. I

THANK YOU EXECUTIVE 4, team netmarble, in Korean, admin group, and friends from Gil. Number one.

‪#‎Heartcold‬ (please no comments about others in this post.)

so we won't got banned for using this kind of bot.
(hope daelus delete comment in front description) lol
